test/TEST-54-CREDS/test.sh

```diff
@@ -4,8 +4,15 @@ set -e
 
 TEST_DESCRIPTION="test credentials"
 NSPAWN_ARGUMENTS="--set-credential=mynspawncredential:strangevalue"
+QEMU_OPTIONS="-fw_cfg  name=opt/io.systemd.credentials/myqemucredential,string=othervalue"
```

Comment:
Please tweak this to something like this:

```suggestion
QEMU_OPTIONS="${QEMU_OPTIONS:-} -fw_cfg  name=opt/io.systemd.credentials/myqemucredential,string=othervalue"
```

(and the same for `NSPAWN_ARGUMENTS=` and `KERNEL_APPEND=`), since in CIs we sometimes tweak these options from the outside as well.